upgrade_test.py

```diff
@@ -657,6 +659,83 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
 
         self.log.info('all nodes were upgraded, and last workaround is verified.')
 
+    def test_generic_cluster_upgrade(self):  # pylint: disable=too-many-locals,too-many-statements
+        """
+        Upgrade half of nodes in the cluster, and start special read workload
+        during the stage. Checksum method is changed to xxhash from Scylla 2.2,
+        we want to use this case to verify the read (cl=ALL) workload works
+        well, upgrade all nodes to new version in the end.
+        """
+
+        # prepare workload (stress_before_upgrade)
+        InfoEvent('Starting stress_before_upgrade - aka prepare step')
+        stress_before_upgrade = self.params.get('stress_before_upgrade')
+        prepare_thread_pool = self.run_stress_thread(stress_cmd=stress_before_upgrade)
+
+        InfoEvent('Waiting for stress_before_upgrade to finish')
+        self.verify_stress_thread(prepare_thread_pool)
+
+        # Starting workload during entire upgrade
+        InfoEvent('Starting stress_during_entire_upgrade workload')
+        stress_during_entire_upgrade = self.params.get('stress_during_entire_upgrade')
+        stress_thread_pool = self.run_stress_thread(stress_cmd=stress_during_entire_upgrade)
+        self.log.info('Sleeping for 60s to let cassandra-stress start before the rollback...')
```

Comment:
Please change `60` to `600`

Reply:
good catch, it's actually to change from 600 to 60.